test/Avro/DeconflictSpec.hs

```diff
@@ -92,3 +94,34 @@ spec = describe "Avro.DeconflictSpec" $ do
       let Right deconflicted = A.deconflict BW.schema'Foo BR.schema'Foo decodedAvro
 
       A.fromAvro deconflicted `shouldBe` Success BR.sampleValue
+
+  describe "Type C" $ do
```

Comment:
The name should say what it's doing I think.

Reply:
It is just a group... Names in `it` are saying something.